src/client/client.go

```diff
@@ -50,7 +54,7 @@ const DefaultMaxConcurrentStreams uint = 100
 
 var (
 	// MaxMsgSize is used to define the GRPC frame size
-	MaxMsgSize = 10 * 1024 * 1024
+	MaxMsgSize = 20 * 1024 * 1024
```

Comment:
In benchmarks I got the best results with a chunk size of `10 MB`, we cut this value in half to account for extra overhead in the message so this change puts us on the observed sweet spot for best performance.